# CENSO

Para calcular el nivel socioecononico voy a usar la siguiente metodología. Tomando:

1. Número de hogares con computadora: Indica la cantidad de hogares que tienen al menos una computadora.

2. Porcentaje de menores de 18 años asistiendo a establecimientos educativos: Calculado como la proporción de menores de 18 años que asisten a algún establecimiento educativo respecto al total de menores de 18 años.

3. Porcentaje de habitantes mayores de 25 años con nivel universitario completo: Calculado como la proporción de habitantes mayores de 25 años con nivel universitario completo respecto al total de habitantes mayores de 25 años.

$IndiceComputadora = \frac{NúmeroDeHogaresConComputadora}{ConComputadora \cup ConHeladera \cup ConCelular \cup ConTeléfono}$

$IndiceEscolaridad = \frac{MenoresDe18Asisten}{MenoresDe18}$

$IndiceEducSuperior = \frac{MayoresDe25Universitarios}{MayoresDe25}$

PC1=0.76120783∗IndiceEducSup+0.10759338∗IndiceEscolaridad+0.63952037∗IndiceCompu

El PC1 arroja un rango de valores entre 0 y 1.32 (aproximadamente) se procede a una normalización para facilitar la lectura de los valores.

$NSE = \frac{PC1}{PC1Max} * 10$

De esta manera todos los valores del índice de NSE de radios censales se expresan en una escala del 0 al 10, donde 10 es el máximo valor de NSE alcanzado por el primer radio censal del ranking y 0 es el mínimo 

(fuente: https://gcba.github.io/tod/nota_metodologica#2-estimacion-del-nivel-socioeconomico-de-un-radio-censal)

Valor Bajo (0-3): Puede indicar un nivel socioeconómico bajo, con posiblemente menos acceso a recursos y oportunidades.

Valor Medio (4-7): Podría sugerir un nivel socioeconómico medio, con un acceso moderado a recursos y oportunidades.

Valor Alto (8-10): Indicaría un nivel socioeconómico relativamente alto, con un buen acceso a recursos y oportunidades.

# Comuna 1


In [29]:
import pandas as pd
import numpy as np
import  matplotlib.pyplot as plt
import  seaborn as sns
from sklearn.decomposition import PCA 

In [33]:
def calcular_indice_computadora(numero_hogares_con_computadora, con_computadora, con_heladera, con_celular, con_telefono):
    indice_computadora = numero_hogares_con_computadora / (con_computadora + con_heladera + con_celular + con_telefono)
    return indice_computadora

# Parámetros para comuna 1
numero_hogares_con_computadora_c1 = 51500
con_computadora_c1 = 51500
con_heladera_c1 = 77558
con_celular_c1 = 73457
con_telefono_c1 = 60208

# Calcular el índice de computadora 
indice_computadora_c1 = calcular_indice_computadora(numero_hogares_con_computadora_c1, con_computadora_c1, con_heladera_c1, con_celular_c1, con_telefono_c1)

print("Índice de Computadora - Comuna 1:", indice_computadora_c1)

Índice de Computadora - Comuna 1: 0.19602394917841223


In [34]:
def calcular_indice_escolaridad(menores_de_18_asisten, menores_de_18):
    indice_escolaridad = menores_de_18_asisten / menores_de_18
    return indice_escolaridad


menores_de_18_asisten_c1 = 2985 + 1991 + 13712 + 3865 + 5164
menores_de_18_c1 = 4162 + 2130 + 13922 + 4010 + 5958

# Calcular el índice de escolaridad para comuna 1
indice_escolaridad_c1 = calcular_indice_escolaridad(menores_de_18_asisten_c1, menores_de_18_c1)

print("Índice de Escolaridad - Comuna 1:", indice_escolaridad_c1)

Índice de Escolaridad - Comuna 1: 0.9183288052481612


In [35]:
def calcular_indice_educacion_superior(mayores_de_25_universitarios, mayores_de_25):
    indice_educacion_superior = mayores_de_25_universitarios / mayores_de_25
    return indice_educacion_superior

# Parámetros para comuna 1
mayores_de_25_universitarios_c1 = 14687
mayores_de_25_c1 = 25118 + 21421 + 121658

# Calcular el índice de educación superior para comuna 1
indice_educacion_superior_c1 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c1, mayores_de_25_c1)

print("Índice de Educación Superior - Caso 1:", indice_educacion_superior_c1)

Índice de Educación Superior - Caso 1: 0.08732022568773522


In [37]:
def calcular_pca(indice_educacion_superior, indice_escolaridad, indice_computadora, n_components=3):
    # Calcula la matriz 
    matriz_correlaciones = np.array([
        [1, indice_educacion_superior, indice_computadora],
        [indice_educacion_superior, 1, indice_escolaridad],
        [indice_computadora, indice_escolaridad, 1]
    ])

    # Realiza el análisis de componentes principales
    pca = PCA(n_components=n_components)
    pca.fit(matriz_correlaciones)

    # Obtiene los componentes principales y la varianza explicada
    componentes_principales = pca.components_
    varianza_explicada = pca.explained_variance_ratio_

    return componentes_principales, varianza_explicada

componentes_principales, varianza_explicada = calcular_pca(indice_educacion_superior_c1, indice_escolaridad_c1, indice_computadora_c1)

print("\nComponentes Principales:")
print(componentes_principales)

print("\nVarianza Explicada:")
print(varianza_explicada)


Componentes Principales:
[[ 0.59696031 -0.60562202 -0.52617522]
 [ 0.46662516 -0.27140543  0.84178385]
 [-0.65260964 -0.74803815  0.1205802 ]]

Varianza Explicada:
[9.92815398e-01 7.18460163e-03 1.77381678e-32]


In [38]:
def calcular_pc1(indice_educacion_superior, indice_escolaridad, indice_computadora):
    # Coeficientes para el cálculo de PC1
    coef_educ_sup = 0.76120783
    coef_escolaridad = 0.10759338
    coef_computadora = 0.63952037

    # Calcula PC1
    pc1 = coef_educ_sup * indice_educacion_superior + coef_escolaridad * indice_escolaridad + coef_computadora * indice_computadora

    return pc1

# Calcula PC1
pc1_resultado = calcular_pc1(indice_educacion_superior_c1, indice_escolaridad_c1, indice_computadora_c1)
print("PC1:", pc1_resultado)

PC1: 0.29063624812632194


In [39]:
def normalizar_pc1(pc1, pc1_max):
    # Normaliza PC1
    nse = (pc1 / pc1_max) * 10
    return nse

pc1_max = varianza_explicada [0]

# Normaliza PC1
nse_resultado = normalizar_pc1(pc1_resultado, pc1_max)
print("NSE:", nse_resultado)

NSE: 2.9273946456024857


El valor normalizado del índice socioeconómico (NSE) obtenido a través del análisis de componentes principales (PCA) representa una medida relativa del nivel socioeconómico, en el caso de la comuna 1 es de 2.9273946456024857 y en esta comuna ganó la presidencia Juntos por el cambio. 

# Comuna 2

In [40]:
# Parámetros para comuna 2
numero_hogares_con_computadora_c2 = 57998
con_computadora_c2 = 57998
con_heladera_c2= 72641
con_celular_c2 = 65253
con_telefono_c2 = 68708

# Calcular el índice de computadora 
indice_computadora_c2 = calcular_indice_computadora(numero_hogares_con_computadora_c2, con_computadora_c2, con_heladera_c2, con_celular_c2, con_telefono_c2)

print("Índice de Computadora - Comuna 1:", indice_computadora_c2)

Índice de Computadora - Comuna 1: 0.2191912320483749


In [41]:
menores_de_18_asisten_c2 = 2109+1188+7719+2173+3447
menores_de_18_c2 = 2276+1211+7762+2209+3636

# Calcular el índice de escolaridad para comuna 2
indice_escolaridad_c2 = calcular_indice_escolaridad(menores_de_18_asisten_c2, menores_de_18_c2)

print("Índice de Escolaridad - Comuna 1:", indice_escolaridad_c2)

Índice de Escolaridad - Comuna 1: 0.9732069732069732


In [42]:
def calcular_indice_educacion_superior(mayores_de_25_universitarios, mayores_de_25):
    indice_educacion_superior = mayores_de_25_universitarios / mayores_de_25
    return indice_educacion_superior

# Parámetros para comuna 2
mayores_de_25_universitarios_c2 = 21323
mayores_de_25_c2 =  21665+16686+9551

# Calcular el índice de educación superior para comuna 2
indice_educacion_superior_c2 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c2, mayores_de_25_c2)

print("Índice de Educación Superior - Caso 2:", indice_educacion_superior_c2)

Índice de Educación Superior - Caso 2: 0.4451379900630454


In [47]:
componentes_principales_2, varianza_explicada_2 = calcular_pca(indice_educacion_superior_c2, indice_escolaridad_c2, indice_computadora_c2)

print("\nComponentes Principales - Comuna 2:")
print(componentes_principales_2)

print("\nVarianza Explicada - Comuna 2:")
print(varianza_explicada_2)



Componentes Principales - Comuna 2:
[[ 0.59084236 -0.46275526 -0.66088038]
 [ 0.7896976   0.49942266  0.35630704]
 [ 0.16517568 -0.73241694  0.66051678]]

Varianza Explicada - Comuna 2:
[9.81581910e-01 1.84180899e-02 2.88114938e-35]


In [52]:
pc2_resultado = calcular_pc1(indice_educacion_superior_c2, indice_escolaridad_c2, indice_computadora_c2)
print("PC2:", pc2_resultado)

PC2: 0.5837304089736925


In [54]:
pc2_max = varianza_explicada_2 [0]
# Normaliza PC2
nse_resultado2 = normalizar_pc1(pc2_resultado, pc2_max)
print("NSE:", nse_resultado2)

NSE: 5.946833401872106


El valor normalizado del índice socioeconómico (NSE) obtenido a través del análisis de componentes principales (PCA) representa una medida relativa del nivel socioeconómico, en el caso de la comuna 1 es de 5.946833401872106 y en esta comuna ganó la presidencia Juntos por el cambio.) Se ubicaria en el nivel medio. 

# Comuna 3

In [55]:
# Parámetros para comuna 3
numero_hogares_con_computadora_c3 = 51631
con_computadora_c3 = 51631
con_heladera_c3 = 76928
con_celular_c3 =69059
con_telefono_c3 = 64202
# Calcular el índice de computadora para comuna 3
indice_computadora_c3 = calcular_indice_computadora(numero_hogares_con_computadora_c3, con_computadora_c3, con_heladera_c3, con_celular_c3, con_telefono_c3)

print("Índice de Computadora - Comuna 3:", indice_computadora_c3)

menores_de_18_asisten_c3 = 3073+1833+12631+3502+4979
menores_de_18_c3 = +3722+1895+12739+3580+5517

# Calcular el índice de escolaridad para comuna 3
indice_escolaridad_c3 = calcular_indice_escolaridad(menores_de_18_asisten_c3, menores_de_18_c3)

print("Índice de Escolaridad - Comuna 3:", indice_escolaridad_c3)

# Parámetros para comuna 3
mayores_de_25_universitarios_c3 = 14818
mayores_de_25_c3 = 21217+17794+114907

# Calcular el índice de educación superior para comuna 3
indice_educacion_superior_c3 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c3, mayores_de_25_c3)

print("Índice de Educación Superior - Comuna 3:", indice_educacion_superior_c3)


Índice de Computadora - Comuna 3: 0.19720036666412039
Índice de Escolaridad - Comuna 3: 0.9477288456634976
Índice de Educación Superior - Comuna 3: 0.09627204095687314


In [56]:
# Calcular PCA para comuna 3
componentes_principales_c3, varianza_explicada_c3 = calcular_pca(indice_educacion_superior_c3, indice_escolaridad_c3, indice_computadora_c3)

print("\nComponentes Principales - Comuna 3:")
print(componentes_principales_c3)

print("\nVarianza Explicada - Comuna 3:")
print(varianza_explicada_c3)

# Calcula PC1 para comuna 3
pc1_resultado_c3 = calcular_pc1(indice_educacion_superior_c3, indice_escolaridad_c3, indice_computadora_c3)
print("PC1 - Comuna 3:", pc1_resultado_c3)

# Normaliza PC1 para comuna 3
pc1_max_c3 = varianza_explicada_c3[0]
nse_resultado_c3 = normalizar_pc1(pc1_resultado_c3, pc1_max_c3)
print("NSE - Comuna 3:", nse_resultado_c3)



Componentes Principales - Comuna 3:
[[ 0.59004514 -0.60570596 -0.53382303]
 [ 0.58916831 -0.12904338  0.79763933]
 [ 0.55202122  0.78515483 -0.28072134]]

Varianza Explicada - Comuna 3:
[9.95861810e-01 4.13819014e-03 9.62138151e-33]
PC1 - Comuna 3: 0.3013660326680605
NSE - Comuna 3: 3.026183248360579


El valor normalizado del índice socioeconómico (NSE) obtenido a través del análisis de componentes principales (PCA) representa una medida relativa del nivel socioeconómico, en el caso de la comuna 1 es de 3.026183248360579 y en esta comuna ganó la presidencia Juntos por el cambio.) Se ubicaria en el nivel bajo.

# Comuna 4

In [57]:
# Parámetros para comuna 4
numero_hogares_con_computadora_c4 = 42164
con_computadora_c4 = 42164
con_heladera_c4 = 73047
con_celular_c4 =65630
con_telefono_c4 = 53996

# Calcular el índice de computadora para comuna 4
indice_computadora_c4 = calcular_indice_computadora(numero_hogares_con_computadora_c4, con_computadora_c4, con_heladera_c4, con_celular_c4, con_telefono_c4)

print("Índice de Computadora - Comuna 4:", indice_computadora_c4)

menores_de_18_asisten_c4 = 4610+2869+20869+5901
menores_de_18_c4 = 5950+2991+21056+6111+9038
# Calcular el índice de escolaridad para comuna 4
indice_escolaridad_c4 = calcular_indice_escolaridad(menores_de_18_asisten_c4, menores_de_18_c4)

print("Índice de Escolaridad - Comuna 4:", indice_escolaridad_c4)

# Parámetros para comuna 4
mayores_de_25_universitarios_c4 = 8401
mayores_de_25_c4 = 24979+18214+120267

# Calcular el índice de educación superior para comuna 4
indice_educacion_superior_c4 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c4, mayores_de_25_c4)

print("Índice de Educación Superior - Comuna 4:", indice_educacion_superior_c4)


Índice de Computadora - Comuna 4: 0.17954581262748204
Índice de Escolaridad - Comuna 4: 0.7586275639037788
Índice de Educación Superior - Comuna 4: 0.05139483665728618


In [58]:
# Calcular PCA para comuna 4
componentes_principales_c4, varianza_explicada_c4 = calcular_pca(indice_educacion_superior_c4, indice_escolaridad_c4, indice_computadora_c4)

print("\nComponentes Principales - Comuna 4:")
print(componentes_principales_c4)

print("\nVarianza Explicada - Comuna 4:")
print(varianza_explicada_c4)

# Calcula PC1 para comuna 4
pc1_resultado_c4 = calcular_pc1(indice_educacion_superior_c4, indice_escolaridad_c4, indice_computadora_c4)
print("PC1 - Comuna 4:", pc1_resultado_c4)

# Normaliza PC1 para comuna 4
pc1_max_c4 = varianza_explicada_c4[0]
nse_resultado_c4 = normalizar_pc1(pc1_resultado_c4, pc1_max_c4)
print("NSE - Comuna 4:", nse_resultado_c4)



Componentes Principales - Comuna 4:
[[ 0.63473931 -0.59912592 -0.48801039]
 [ 0.17086582 -0.50708852  0.84478761]
 [-0.75359862 -0.6196042  -0.21949887]]

Varianza Explicada - Comuna 4:
[9.55599771e-01 4.44002295e-02 2.44782337e-32]
PC1 - Comuna 4: 0.2355686603701488
NSE - Comuna 4: 2.46513935684701


 El valor normalizado del índice socioeconómico (NSE) obtenido a través del análisis de componentes principales (PCA) representa una medida relativa del nivel socioeconómico, en el caso de la comuna 1 es de 2.46513935684701 y en esta comuna ganó la presidencia Unión por la patria. Se ubicaria en el nivel bajo.

# Comuna 5

In [65]:
# Parámetros para comuna 5
numero_hogares_con_computadora_c5 = 54204
con_computadora_c5 = 54204
con_heladera_c5 = 75319
con_celular_c5 = 67096
con_telefono_c5 =66779
# Calcular el índice de computadora para comuna 5
indice_computadora_c5 = calcular_indice_computadora(numero_hogares_con_computadora_c5, con_computadora_c5, con_heladera_c5, con_celular_c5, con_telefono_c5)

print("Índice de Computadora - Comuna 5:", indice_computadora_c5)

menores_de_18_asisten_c5 = 3072+1746+11875+3166+4786
menores_de_18_c5 = 3430+1776+11950+3228+5160

# Calcular el índice de escolaridad para comuna 5
indice_escolaridad_c5 = calcular_indice_escolaridad(menores_de_18_asisten_c5, menores_de_18_c5)

print("Índice de Escolaridad - Comuna 5:", indice_escolaridad_c5)

# Parámetros para comuna 5
mayores_de_25_universitarios_c5 = 14170
mayores_de_25_c5 = 17528+15948+114409
# Calcular el índice de educación superior para comuna 5
indice_educacion_superior_c5 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c5, mayores_de_25_c5)

print("Índice de Educación Superior - Comuna 5:", indice_educacion_superior_c5)


Índice de Computadora - Comuna 5: 0.20578743954016354
Índice de Escolaridad - Comuna 5: 0.9648058252427184
Índice de Educación Superior - Comuna 5: 0.09581769618284477


In [66]:
# Calcular PCA para comuna 5
componentes_principales_c5, varianza_explicada_c5 = calcular_pca(indice_educacion_superior_c5, indice_escolaridad_c5, indice_computadora_c5)

print("\nComponentes Principales - Comuna 5:")
print(componentes_principales_c5)

print("\nVarianza Explicada - Comuna 5:")
print(varianza_explicada_c5)

# Calcula PC1 para comuna 5
pc1_resultado_c5 = calcular_pc1(indice_educacion_superior_c5, indice_escolaridad_c5, indice_computadora_c5)
print("PC1 - Comuna 5:", pc1_resultado_c5)

# Normaliza PC1 para comuna 5
pc1_max_c5 = varianza_explicada_c5[0]
nse_resultado_c5 = normalizar_pc1(pc1_resultado_c5, pc1_max_c5)
print("NSE - Comuna 5:", nse_resultado_c5)



Componentes Principales - Comuna 5:
[[ 0.58625696 -0.61017005 -0.53291208]
 [ 0.70382015  0.05786653  0.70801742]
 [ 0.40117325  0.7901544  -0.46337463]]

Varianza Explicada - Comuna 5:
[9.96438750e-01 3.56124963e-03 3.28671347e-33]
PC1 - Comuna 5: 0.308349159844574
NSE - Comuna 5: 3.0945119279067197


El valor normalizado del índice socioeconómico (NSE) obtenido a través del análisis de componentes principales (PCA) representa una medida relativa del nivel socioeconómico, en el caso de la comuna 1 es de 3.0945119279067197 y en esta comuna ganó la presidencia Juntos por el cambio. Se ubicaria en el nivel bajo.

# Comuna 6

In [67]:
# Parámetros para comuna 6
numero_hogares_con_computadora_c6 = 56637
con_computadora_c6 = 56637  
con_heladera_c6 = 74619  
con_celular_c6 = 66241 
con_telefono_c6 = 69116

# Calcular el índice de computadora para comuna 6
indice_computadora_c6 = calcular_indice_computadora(numero_hogares_con_computadora_c6, con_computadora_c6, con_heladera_c6, con_celular_c6, con_telefono_c6)

print("Índice de Computadora - Comuna 6:", indice_computadora_c6)

menores_de_18_asisten_c6 = 3241+1707+11237+2952+4530
menores_de_18_c6 = 3467+1732+11302+3007+4785

# Calcular el índice de escolaridad para comuna 6
indice_escolaridad_c6 = calcular_indice_escolaridad(menores_de_18_asisten_c6, menores_de_18_c6)

print("Índice de Escolaridad - Comuna 6:", indice_escolaridad_c6)

# Parámetros para comuna 6
mayores_de_25_universitarios_c6 = 13951
mayores_de_25_c6 = 15426+14191+116538

# Calcular el índice de educación superior para comuna 6
indice_educacion_superior_c6 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c6, mayores_de_25_c6)

print("Índice de Educación Superior - Comuna 6:", indice_educacion_superior_c6)


Índice de Computadora - Comuna 6: 0.21243150183974524
Índice de Escolaridad - Comuna 6: 0.9742312600337546
Índice de Educación Superior - Comuna 6: 0.09545345694639253


In [68]:
# Calcular PCA para comuna 6
componentes_principales_c6, varianza_explicada_c6 = calcular_pca(indice_educacion_superior_c6, indice_escolaridad_c6, indice_computadora_c6)

print("\nComponentes Principales - Comuna 6:")
print(componentes_principales_c6)

print("\nVarianza Explicada - Comuna 6:")
print(varianza_explicada_c6)

# Calcula PC1 para comuna 6
pc1_resultado_c6 = calcular_pc1(indice_educacion_superior_c6, indice_escolaridad_c6, indice_computadora_c6)
print("PC1 - Comuna 6:", pc1_resultado_c6)

# Normaliza PC1 para comuna 6
pc1_max_c6 = varianza_explicada_c6[0]
nse_resultado_c6 = normalizar_pc1(pc1_resultado_c6, pc1_max_c6)
print("NSE - Comuna 6:", nse_resultado_c6)


Componentes Principales - Comuna 6:
[[ 0.58421015 -0.61326431 -0.53160643]
 [ 0.75636743  0.17388999  0.63061127]
 [ 0.29429035  0.77049929 -0.56544145]]

Varianza Explicada - Comuna 6:
[9.96442270e-01 3.55773038e-03 1.28227857e-32]
PC1 - Comuna 6: 0.31333502565306204
NSE - Comuna 6: 3.1445376737510666


El valor normalizado del índice socioeconómico (NSE) obtenido a través del análisis de componentes principales (PCA) representa una medida relativa del nivel socioeconómico, en el caso de la comuna 1 es de 3.1445376737510666 y en esta comuna ganó la presidencia Juntos por el cambio. Se ubicaria en el nivel bajo.

# Comuna 7

In [ ]:
# Parámetros para comuna 7
numero_hogares_con_computadora_c7 = ...  # Completa con los datos correspondientes para la Comuna 7
con_computadora_c7 = ...  # Completa con los datos correspondientes para la Comuna 7
con_heladera_c7 = ...  # Completa con los datos correspondientes para la Comuna 7
con_celular_c7 = ...  # Completa con los datos correspondientes para la Comuna 7
con_telefono_c7 = ...  # Completa con los datos correspondientes para la Comuna 7

# Calcular el índice de computadora para comuna 7
indice_computadora_c7 = calcular_indice_computadora(numero_hogares_con_computadora_c7, con_computadora_c7, con_heladera_c7, con_celular_c7, con_telefono_c7)

print("Índice de Computadora - Comuna 7:", indice_computadora_c7)

menores_de_18_asisten_c7 = ...  # Completa con los datos correspondientes para la Comuna 7
menores_de_18_c7 = ...  # Completa con los datos correspondientes para la Comuna 7

# Calcular el índice de escolaridad para comuna 7
indice_escolaridad_c7 = calcular_indice_escolaridad(menores_de_18_asisten_c7, menores_de_18_c7)

print("Índice de Escolaridad - Comuna 7:", indice_escolaridad_c7)

# Parámetros para comuna 7
mayores_de_25_universitarios_c7 = ...  # Completa con los datos correspondientes para la Comuna 7
mayores_de_25_c7 = ...  # Completa con los datos correspondientes para la Comuna 7

# Calcular el índice de educación superior para comuna 7
indice_educacion_superior_c7 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c7, mayores_de_25_c7)

print("Índice de Educación Superior - Comuna 7:", indice_educacion_superior_c7)


# Comuna 8 

In [ ]:
# Parámetros para comuna 8
numero_hogares_con_computadora_c8 = ...  # Completa con los datos correspondientes para la Comuna 8
con_computadora_c8 = ...  # Completa con los datos correspondientes para la Comuna 8
con_heladera_c8 = ...  # Completa con los datos correspondientes para la Comuna 8
con_celular_c8 = ...  # Completa con los datos correspondientes para la Comuna 8
con_telefono_c8 = ...  # Completa con los datos correspondientes para la Comuna 8

# Calcular el índice de computadora para comuna 8
indice_computadora_c8 = calcular_indice_computadora(numero_hogares_con_computadora_c8, con_computadora_c8, con_heladera_c8, con_celular_c8, con_telefono_c8)

print("Índice de Computadora - Comuna 8:", indice_computadora_c8)

menores_de_18_asisten_c8 = ...  # Completa con los datos correspondientes para la Comuna 8
menores_de_18_c8 = ...  # Completa con los datos correspondientes para la Comuna 8

# Calcular el índice de escolaridad para comuna 8
indice_escolaridad_c8 = calcular_indice_escolaridad(menores_de_18_asisten_c8, menores_de_18_c8)

print("Índice de Escolaridad - Comuna 8:", indice_escolaridad_c8)

# Parámetros para comuna 8
mayores_de_25_universitarios_c8 = ...  # Completa con los datos correspondientes para la Comuna 8
mayores_de_25_c8 = ...  # Completa con los datos correspondientes para la Comuna 8

# Calcular el índice de educación superior para comuna 8
indice_educacion_superior_c8 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c8, mayores_de_25_c8)

print("Índice de Educación Superior - Comuna 8:", indice_educacion_superior_c8)


# Comuna 9

In [ ]:
# Parámetros para comuna 9
numero_hogares_con_computadora_c9 = ...  # Completa con los datos correspondientes para la Comuna 9
con_computadora_c9 = ...  # Completa con los datos correspondientes para la Comuna 9
con_heladera_c9 = ...  # Completa con los datos correspondientes para la Comuna 9
con_celular_c9 = ...  # Completa con los datos correspondientes para la Comuna 9
con_telefono_c9 = ...  # Completa con los datos correspondientes para la Comuna 9

# Calcular el índice de computadora para comuna 9
indice_computadora_c9 = calcular_indice_computadora(numero_hogares_con_computadora_c9, con_computadora_c9, con_heladera_c9, con_celular_c9, con_telefono_c9)

print("Índice de Computadora - Comuna 9:", indice_computadora_c9)

menores_de_18_asisten_c9 = ...  # Completa con los datos correspondientes para la Comuna 9
menores_de_18_c9 = ...  # Completa con los datos correspondientes para la Comuna 9

# Calcular el índice de escolaridad para comuna 9
indice_escolaridad_c9 = calcular_indice_escolaridad(menores_de_18_asisten_c9, menores_de_18_c9)

print("Índice de Escolaridad - Comuna 9:", indice_escolaridad_c9)

# Parámetros para comuna 9
mayores_de_25_universitarios_c9 = ...  # Completa con los datos correspondientes para la Comuna 9
mayores_de_25_c9 = ...  # Completa con los datos correspondientes para la Comuna 9

# Calcular el índice de educación superior para comuna 9
indice_educacion_superior_c9 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c9, mayores_de_25_c9)

print("Índice de Educación Superior - Comuna 9:", indice_educacion_superior_c9)


# Comuna 10

In [ ]:
# Parámetros para comuna 10
numero_hogares_con_computadora_c10 = ...  # Completa con los datos correspondientes para la Comuna 10
con_computadora_c10 = ...  # Completa con los datos correspondientes para la Comuna 10
con_heladera_c10 = ...  # Completa con los datos correspondientes para la Comuna 10
con_celular_c10 = ...  # Completa con los datos correspondientes para la Comuna 10
con_telefono_c10 = ...  # Completa con los datos correspondientes para la Comuna 10

# Calcular el índice de computadora para comuna 10
indice_computadora_c10 = calcular_indice_computadora(numero_hogares_con_computadora_c10, con_computadora_c10, con_heladera_c10, con_celular_c10, con_telefono_c10)

print("Índice de Computadora - Comuna 10:", indice_computadora_c10)

menores_de_18_asisten_c10 = ...  # Completa con los datos correspondientes para la Comuna 10
menores_de_18_c10 = ...  # Completa con los datos correspondientes para la Comuna 10

# Calcular el índice de escolaridad para comuna 10
indice_escolaridad_c10 = calcular_indice_escolaridad(menores_de_18_asisten_c10, menores_de_18_c10)

print("Índice de Escolaridad - Comuna 10:", indice_escolaridad_c10)

# Parámetros para comuna 10
mayores_de_25_universitarios_c10 = ...  # Completa con los datos correspondientes para la Comuna 10
mayores_de_25_c10 = ...  # Completa con los datos correspondientes para la Comuna 10

# Calcular el índice de educación superior para comuna 10
indice_educacion_superior_c10 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c10, mayores_de_25_c10)

print("Índice de Educación Superior - Comuna 10:", indice_educacion_superior_c10)


# Comuna 11

In [ ]:
# Parámetros para comuna 11
numero_hogares_con_computadora_c11 = ...  # Completa con los datos correspondientes para la Comuna 11
con_computadora_c11 = ...  # Completa con los datos correspondientes para la Comuna 11
con_heladera_c11 = ...  # Completa con los datos correspondientes para la Comuna 11
con_celular_c11 = ...  # Completa con los datos correspondientes para la Comuna 11
con_telefono_c11 = ...  # Completa con los datos correspondientes para la Comuna 11

# Calcular el índice de computadora para comuna 11
indice_computadora_c11 = calcular_indice_computadora(numero_hogares_con_computadora_c11, con_computadora_c11, con_heladera_c11, con_celular_c11, con_telefono_c11)

print("Índice de Computadora - Comuna 11:", indice_computadora_c11)

menores_de_18_asisten_c11 = ...  # Completa con los datos correspondientes para la Comuna 11
menores_de_18_c11 = ...  # Completa con los datos correspondientes para la Comuna 11

# Calcular el índice de escolaridad para comuna 11
indice_escolaridad_c11 = calcular_indice_escolaridad(menores_de_18_asisten_c11, menores_de_18_c11)

print("Índice de Escolaridad - Comuna 11:", indice_escolaridad_c11)

# Parámetros para comuna 11
mayores_de_25_universitarios_c11 = ...  # Completa con los datos correspondientes para la Comuna 11
mayores_de_25_c11 = ...  # Completa con los datos correspondientes para la Comuna 11

# Calcular el índice de educación superior para comuna 11
indice_educacion_superior_c11 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c11, mayores_de_25_c11)

print("Índice de Educación Superior - Comuna 11:", indice_educacion_superior_c11)


# Comuna 12 

In [59]:
# Parámetros para comuna 11
numero_hogares_con_computadora_c11 = ...  # Completa con los datos correspondientes para la Comuna 11
con_computadora_c11 = ...  # Completa con los datos correspondientes para la Comuna 11
con_heladera_c11 = ...  # Completa con los datos correspondientes para la Comuna 11
con_celular_c11 = ...  # Completa con los datos correspondientes para la Comuna 11
con_telefono_c11 = ...  # Completa con los datos correspondientes para la Comuna 11

# Calcular el índice de computadora para comuna 11
indice_computadora_c11 = calcular_indice_computadora(numero_hogares_con_computadora_c11, con_computadora_c11, con_heladera_c11, con_celular_c11, con_telefono_c11)

print("Índice de Computadora - Comuna 11:", indice_computadora_c11)

menores_de_18_asisten_c11 = ...  # Completa con los datos correspondientes para la Comuna 11
menores_de_18_c11 = ...  # Completa con los datos correspondientes para la Comuna 11

# Calcular el índice de escolaridad para comuna 11
indice_escolaridad_c11 = calcular_indice_escolaridad(menores_de_18_asisten_c11, menores_de_18_c11)

print("Índice de Escolaridad - Comuna 11:", indice_escolaridad_c11)

# Parámetros para comuna 11
mayores_de_25_universitarios_c11 = ...  # Completa con los datos correspondientes para la Comuna 11
mayores_de_25_c11 = ...  # Completa con los datos correspondientes para la Comuna 11

# Calcular el índice de educación superior para comuna 11
indice_educacion_superior_c11 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c11, mayores_de_25_c11)

print("Índice de Educación Superior - Comuna 11:", indice_educacion_superior_c11)


TypeError: unsupported operand type(s) for +: 'ellipsis' and 'ellipsis'

# Comuna 13

In [ ]:
# Parámetros para comuna 13
numero_hogares_con_computadora_c13 = ...  # Completa con los datos correspondientes para la Comuna 13
con_computadora_c13 = ...  # Completa con los datos correspondientes para la Comuna 13
con_heladera_c13 = ...  # Completa con los datos correspondientes para la Comuna 13
con_celular_c13 = ...  # Completa con los datos correspondientes para la Comuna 13
con_telefono_c13 = ...  # Completa con los datos correspondientes para la Comuna 13

# Calcular el índice de computadora para comuna 13
indice_computadora_c13 = calcular_indice_computadora(numero_hogares_con_computadora_c13, con_computadora_c13, con_heladera_c13, con_celular_c13, con_telefono_c13)

print("Índice de Computadora - Comuna 13:", indice_computadora_c13)

menores_de_18_asisten_c13 = ...  # Completa con los datos correspondientes para la Comuna 13
menores_de_18_c13 = ...  # Completa con los datos correspondientes para la Comuna 13

# Calcular el índice de escolaridad para comuna 13
indice_escolaridad_c13 = calcular_indice_escolaridad(menores_de_18_asisten_c13, menores_de_18_c13)

print("Índice de Escolaridad - Comuna 13:", indice_escolaridad_c13)

# Parámetros para comuna 13
mayores_de_25_universitarios_c13 = ...  # Completa con los datos correspondientes para la Comuna 13
mayores_de_25_c13 = ...  # Completa con los datos correspondientes para la Comuna 13

# Calcular el índice de educación superior para comuna 13
indice_educacion_superior_c13 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c13, mayores_de_25_c13)

print("Índice de Educación Superior - Comuna 13:", indice_educacion_superior_c13)


# Comuna 14

In [ ]:
# Parámetros para comuna 14
numero_hogares_con_computadora_c14 = ...  # Completa con los datos correspondientes para la Comuna 14
con_computadora_c14 = ...  # Completa con los datos correspondientes para la Comuna 14
con_heladera_c14 = ...  # Completa con los datos correspondientes para la Comuna 14
con_celular_c14 = ...  # Completa con los datos correspondientes para la Comuna 14
con_telefono_c14 = ...  # Completa con los datos correspondientes para la Comuna 14

# Calcular el índice de computadora para comuna 14
indice_computadora_c14 = calcular_indice_computadora(numero_hogares_con_computadora_c14, con_computadora_c14, con_heladera_c14, con_celular_c14, con_telefono_c14)

print("Índice de Computadora - Comuna 14:", indice_computadora_c14)

menores_de_18_asisten_c14 = ...  # Completa con los datos correspondientes para la Comuna 14
menores_de_18_c14 = ...  # Completa con los datos correspondientes para la Comuna 14

# Calcular el índice de escolaridad para comuna 14
indice_escolaridad_c14 = calcular_indice_escolaridad(menores_de_18_asisten_c14, menores_de_18_c14)

print("Índice de Escolaridad - Comuna 14:", indice_escolaridad_c14)

# Parámetros para comuna 14
mayores_de_25_universitarios_c14 = ...  # Completa con los datos correspondientes para la Comuna 14
mayores_de_25_c14 = ...  # Completa con los datos correspondientes para la Comuna 14

# Calcular el índice de educación superior para comuna 14
indice_educacion_superior_c14 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c14, mayores_de_25_c14)

print("Índice de Educación Superior - Comuna 14:", indice_educacion_superior_c14)


# Comuna 15

In [ ]:
# Parámetros para comuna 15
numero_hogares_con_computadora_c15 = ...  # Completa con los datos correspondientes para la Comuna 15
con_computadora_c15 = ...  # Completa con los datos correspondientes para la Comuna 15
con_heladera_c15 = ...  # Completa con los datos correspondientes para la Comuna 15
con_celular_c15 = ...  # Completa con los datos correspondientes para la Comuna 15
con_telefono_c15 = ...  # Completa con los datos correspondientes para la Comuna 15

# Calcular el índice de computadora para comuna 15
indice_computadora_c15 = calcular_indice_computadora(numero_hogares_con_computadora_c15, con_computadora_c15, con_heladera_c15, con_celular_c15, con_telefono_c15)

print("Índice de Computadora - Comuna 15:", indice_computadora_c15)

menores_de_18_asisten_c15 = ...  # Completa con los datos correspondientes para la Comuna 15
menores_de_18_c15 = ...  # Completa con los datos correspondientes para la Comuna 15

# Calcular el índice de escolaridad para comuna 15
indice_escolaridad_c15 = calcular_indice_escolaridad(menores_de_18_asisten_c15, menores_de_18_c15)

print("Índice de Escolaridad - Comuna 15:", indice_escolaridad_c15)

# Parámetros para comuna 15
mayores_de_25_universitarios_c15 = ...  # Completa con los datos correspondientes para la Comuna 15
mayores_de_25_c15 = ...  # Completa con los datos correspondientes para la Comuna 15

# Calcular el índice de educación superior para comuna 15
indice_educacion_superior_c15 = calcular_indice_educacion_superior(mayores_de_25_universitarios_c15, mayores_de_25_c15)

print("Índice de Educación Superior - Comuna 15:", indice_educacion_superior_c15)
